### ISSUE TITLE
revanced.to: breakage

### ISSUE THREAD

**Issue by daniels220** [uBlockOrigin/uAssets#23569]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://revanced.to/
```

### Description

ReVanced seems to have two websites, revanced.app as seen in the filter list but also revanced.to. The filters need to specify `domain=~revanced.app|~revanced.to` or revanced.to breaks with all CSS and JS blocked.

### Other extensions used

Lots, but none relevant as the issue is very obviously with the filter and disabling just the one filter fixes the site.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<img width="902" alt="Screenshot 2024-05-05 at 12 32 56 AM" src="https://github.com/uBlockOrigin/uAssets/assets/1410866/a4e81604-9a9c-4d0c-a5d9-70ffd21173dc">

<img width="1952" alt="Screenshot 2024-05-05 at 12 33 17 AM" src="https://github.com/uBlockOrigin/uAssets/assets/1410866/2a7e0a30-fc35-4c85-8462-6d51df8ed37f">

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.57.0
Chromium: 124
filterset (summary):
 network: 162593
 cosmetic: 141644
 scriptlet: 48168
 html: 0
listset (total-discarded, last-updated):
 removed:
  easylist: null
  plowe-0: null
 added:
  adguard-cookies: 29435-51, 2d.42m
  adguard-generic: 81475-5018, 2d.41m
  adguard-mobile-app-banners: 5186-7, 2d.41m
  adguard-other-annoyances: 14201-65, 1d.8h.26m
  adguard-popup-overlays: 27272-980, 1d.8h.26m
  adguard-social: 22890-156, 2d.42m
  adguard-spyware: 88706-31663, 2d.41m
  adguard-spyware-url: 1507-119, 2d.41m
  block-lan: 61-0, 19d.23m
  ublock-annoyances: 6967-1119, 49m Δ
  ublock-cookies-adguard: 1086-0, 8h.36m
 default:
  user-filters: 0-0, never
  easyprivacy: 50859-23, 49m Δ
  ublock-badware: 8528-0, 49m Δ
  ublock-filters: 37849-620, 49m Δ
  ublock-privacy: 881-0, 49m Δ
  ublock-quick-fixes: 272-0, 49m Δ
  ublock-unbreak: 2268-1, 49m Δ
  urlhaus-1: 12835-0, 7h.18m
filterset (user): [empty]
trustedset:
 added: [array of 12 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 891 ms (selfie)
 maxAssetCacheWait: 110 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 6
 network:
  github.com: 6
```
</details>

**Comment by JobcenterTycoon** [comment]
The download link is real but the site contains malicious ads. Pretty sure its not a official site, the site owner just want to get a piece of the cake.

**Comment by daniels220** [comment]
I stand corrected, I suppose. I did wonder a little—I've seen things like this before that AFAICT weren't malicious, but the thought did cross my mind that something fishy might be going on.

Ideally there would be better UX for things like this—right now it just looks like "this website is broken, and disabling uBlock fixes it".

**Comment by erikk333** [comment]
.to is not managed by the revanced project.

**Comment by erikk333** [comment]
![Screenshot_2024-05-06-17-18-44-674_app revanced manager flutter](https://github.com/uBlockOrigin/uAssets/assets/147526653/ae20a38a-f1e7-4cce-812a-176b6692fbfe)
This are all revanced's social media accounts and online things. The .to website also refers to a telegram that is called revanced.io, also not Linked in tbe screenshot above. The team is pretty clear about having one website.  Then there is the fact that at https://revanced.to/faq under the question what is revanced.to? is explicitly stated that It is *not* affiliated to revanced . They also share third party magisk modules and pre build apps from j-hc and Noname exe . Then later on in the faq is the last question:

Why did revanced.io change the domain extension?
We changed the domain extension due to complaints we received from the official Revanced team. Kindly visit the update page to know more info

tl;Dr

Revanced.to is a continuation of revanced.io it's not official and prior domain got suspended due to comaplains
